scripts/circle-test-backend.sh

```diff
@@ -1,20 +1,12 @@
 #!/bin/bash
-function exit_if_fail {
-    command=$@
-    echo "Executing '$command'"
-    eval $command
-    rc=$?
-    if [ $rc -ne 0 ]; then
-        echo "'$command' returned $rc."
-        exit $rc
-    fi
-}
+
+source "$(dirname "$0")/helpers/exit-if-fail.sh"
```

Comment:
Been meaning to get this fixed for ages. Great to finally get this one taken care of!

Reply:
:))